<?xml version="1.0"?>
<launch>

  <!-- this node detects grass -->
  <node pkg="rt_road_detection" name="sample_grass_detector" type="sample_grass_detector_node" output="screen" ns="detectors">
   <remap from="rgb_in" to="/stereo/left/image_color"/>
   <remap from="det_out" to="/detectors/not_road/sample_grass"/>
  </node>

</launch>
